apps/Extism/count-vowels/CountVowels.v3

```diff
@@ -0,0 +1,77 @@
+// `defaultVowels` represents the default set of vowels
```

Comment:
Virgil style is generally to use tabs instead of spaces for indentation.

Reply:
A standard formatter like Go or MoonBit would be fantastic. 
I'm sure it's probably on the roadmap.